packages/material-components-web/README.md

```diff
@@ -18,19 +18,23 @@ npm install material-components-web
 ```
 
 ### Including the Javascript
+ 
+### Import TypeScript Module
```

Comment:
Let's also remove this. We are intentionally remaining with plain JS, since we don't want to confuse users that are not on TS.